packages/documentation/src/content/docs/overview/concepts/accounting.mdx

```diff
@@ -1,139 +1,261 @@
+| `assetScale` | Integer | Difference in order of magnitude between the standard unit and a fractional unit | `2` |
+
+</div>
+
+To convert an asset’s value into an amount that’s easier to interpret, apply the following formula.
+
+$\frac{value}{10^{assetScale}}$ = *currencyAmount*
+
+Using the example data from the table above, the formula looks like this:
+
+$\frac{10000}{10^2} =\$100.00 USD
+
+## Accounts
+ 
+Rafiki uses a combination of liquidity and settlement accounts to track the amounts available to fund transactions. Rafiki does not physically hold funds in each account. Instead, it uses <LinkOut href="https://en.wikipedia.org/wiki/Double-entry_bookkeeping">double-entry accounting</LinkOut> to record the transactions. The actual settlement of amounts owed, in which funds are physically exchanged, occurs outside of both Rafiki and the Interledger Protocol. 
 
 ### Liquidity accounts
 
-A liquidity account holds a non-negative balance, with Rafiki ensuring that total debits do not exceed total credits.
+Liquidity accounts are used to track deposits, withdrawals, and transfers that occur during the course of a transaction. Liquidity accounts are provided for assets, peers, and payments.
 
-There is one liquidity account for each of the following resources:
+Liquidity accounts hold either a zero or a positive balance. Rafiki ensures that the total debits to a liquidity account will not exceed the account's total credits.
 
-- Asset
-- Peer
-- Wallet address
-- Incoming payment
-- Outgoing payment
+<div class="overflow-table" style="border-bottom: 1px solid var(--sl-color-gray-6);">
 
-### Settlement accounts
+| Account type | What the account represents | Number of accounts |
+| ------------------------- | --------------------------- | ------------------ |
+| [Asset liquidity](#asset-liquidity-accounts) | The value, denominated in a given asset, that Rafiki has at its disposal for sending or forwarding ILP packets | One per asset |
+| [Peer liquidity](#peer-liquidity-accounts) | The credit line, denominated in the asset of your peering relationship, that you extend to a peer | One per peer |
+| [Incoming payment liquidity](#incoming-payment-liquidity-accounts) | The value received from a completed incoming payment | One per incoming payment |
+| [Outgoing payment liquidity](#outgoing-payment-liquidity-accounts) | The value that Rafiki will attempt to send in an outgoing payment | One per outgoing payment |
+| [Wallet address liquidity](#wallet-address-liquidity-accounts) | The value of an incoming payment received to a wallet address via either SPSP or Web Monetization | One per wallet address |
+
+</div>
+
+#### Asset liquidity accounts
+
+Asset liquidity ensures there is enough value available to handle a foreign exchange transaction between you and a peer. You have one asset liquidity account for each asset you transact in.
```

Comment:
Could also just be within the same rafiki instance as well

Reply:
Slightly revised the overview for this subsection